src/components/HomePageProjects/index.tsx

```diff
@@ -1,42 +1,49 @@
-import Link from '@docusaurus/Link';
-import React from 'react';
-import clsx from 'clsx';
-import styles from './styles.module.css';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Link from "@docusaurus/Link";
```

Comment:
not a blocker but looks like I need to adjust the linter config.

Reply:
Yeah, it wanted to change a bunch of stuff on save.